src/api-reference/user-provisioning/spend/v4.spend-user-provisioning.md

```diff
@@ -44,121 +58,121 @@ POST https://www.us.api.concursolutions.com/provisioning/v4/Bulk/
 
 ##### Parameters
 
-Name|Type|Format|Description
----|---|---|---
-`-`|`-`|-|**Required** -
-`-`|`-`|-|-
+| Name | Type | Format | Description    |
```

Comment:
Is this supposed to be a table with empty content?

Reply:
Very weird. I think we just want to get rid of this right?

Reply:
yeah, we should get rid of all tables w/o any content.